include/openPMD/IO/AccessType.hpp

```diff
@@ -23,12 +23,12 @@
 
 namespace openPMD
 {
-/** File access permissions to use during IO.
- */
-enum class AccessType
-{
-    READ_ONLY,
-    READ_WRITE,
-    CREATE
-};  //AccessType
-} // openPMD
+    /** File access mode to use during IO.
+     */
+    enum class AccessType
+    {
+        READ_ONLY,  //!< open series as read-only
```

Comment:
One more piece of information that might be useful to the user:
`READ_ONLY` raises an Error when no matching files exist, while the other two don't.